assets/js/components/OutfrontTakeoverTool/AlertWizard/AlertWizard.tsx

```diff
@@ -14,8 +14,7 @@ import WizardSidebar from "./WizardSidebar";
 import { svgLongSide, svgScale, svgShortSide } from "../../../constants/misc";
 import { matchStation } from "../../../util";
 
-import parseISO from "date-fns/parseISO";
-import differenceInHours from "date-fns/differenceInHours";
+import { differenceInHours, parseISO } from "date-fns";
```

Comment:
Earlier in this PR `date-fns` is removed as an explicit dependency, but here we are indeed depending on it in our own code, so probably we should add it back?

Reply:
Hmm, very surprised I didn't see any errors after removing it. It must be indirectly imported by something else. Will definitely add it back.

Reply:
It definitely is, since in the commit where it was removed, `package-lock` has a diff where it updates the version of the package rather than removing it (indicating our version constraint was more restrictive than whatever package has it as a dependency, I suppose).